### PR TITLE
Download QCOW2 images from S3

### PR DESCRIPTION
**Pull request by usamasaqib**
### What does this PR do?
Updates the `vmconfig.json` file to specify image source as an S3 URL.

### Motivation
https://github.com/DataDog/test-infra-definitions/pull/221

To summarize, this should help reduce the time of the environment setup step as the number of qcow2 images to download increases. 

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: test/new-e2e/system-probe/config/vmconfig.json [hunk 1]
```diff
@@ -1,52 +1,56 @@
 {
-   "vmsets": [
+    "vmsets": [
         {
-            "name": "ubuntu_x86_64",
+            "name": "distro_distro_x86_64",
```
Comment (brycekahle) [inline test/new-e2e/system-probe/config/vmconfig.json:4]: why `distro_distro`?

File: test/new-e2e/system-probe/config/vmconfig.json [hunk 2]
```diff
@@ -1,52 +1,56 @@
...
                 {
                     "dir": "focal-server-cloudimg-arm64.qcow2",
-                    "tag": "focal_20",
-                    "image_source": "file:///home/kernel-version-testing/rootfs/focal-server-cloudimg-arm64.qcow2"
+                    "tag": "focal",
+                    "image_source": "https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs/focal-server-cloudimg-arm64.qcow2"
                 },
                 {
-                    "dir": "bionic-server-cloudimg-arm64.qcow2",
-                    "tag": "bionic_18",
```
Comment (brycekahle) [inline test/new-e2e/system-probe/config/vmconfig.json:44]: No more bionic?

Comment (usamasaqib) [reply test/new-e2e/system-probe/config/vmconfig.json:44]: We are not supporting arm64 bionic.

**Review by brycekahle — APPROVED** [review]
Looks good once the `distro_distro` bug is fixed.

**Review by KSerrania — APPROVED** [review]
LGTM for Agent Platform files. The CI failures are unrelated (they are due to a bad `curl` release in the Ubuntu security channel from yesterday).

**Pull request merged**